packages/hash/frontend/src/components/Link.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from "react";
```

Comment:
If we're adding this component, I think we should also add the eslint that ban importing the `Link` component from `@mui/material` and `next`/`next/link`. You can [take these directly from the BP repo](https://github.com/blockprotocol/blockprotocol/blob/a38326979c47a74e70c987ddf5f5c65cb2b29b01/.eslintrc.js#L228-L249)

Reply:
We probably need to configure these to be warnings instead of errors though, to prevent you from having to make significant number of changes as part of this PR